### PR TITLE
[MAR-3392] Training

### PR DESCRIPTION
**Pull request by malmas18**

### CODE REVIEW

File: ui-automated-tests/src/flows/brief/training.ts [hunk 1]
```diff
@@ -1,235 +1,133 @@
...
+  await utils.type("start_date", { numberOfCharacters: 150 });
+  await utils.type("contract_length-label", { numberOfCharacters: 50 });
+  await utils.type("contract_extensions", { numberOfCharacters: 50 });
+  await utils.type("budget_range", { numberOfWords: 150 });
   await clickSaveContinue();
 };
 
-const fillCulturalFitCriteria = async () => {
-  await utils.type("input-culturalFitCriteria-1", { numberOfWords: 10 });
+const fillEvaluationCriteria = async () => {
```
Comment (resh-ie) [inline ui-automated-tests/src/flows/brief/training.ts:86]: check and select for desirable criteria
_warning: Desirable weightings must add up to 100%_

Comment (malmas18) [reply ui-automated-tests/src/flows/brief/training.ts:86]: updated

File: ui-automated-tests/src/tests/briefResponse/training.ts [hunk 2]
```diff
@@ -2,30 +2,30 @@ import create from "../../flows/brief/training";
...
       title,
     });
   });
 
-  it(`should be able to respond to training brief`, async () => {
+  it("should be able to respond training brief", async () => {
     await sellerLogin();
     await navigate();
     await selectBrief(title);
+    await sleep(1000);
```
Comment (resh-ie) [inline ui-automated-tests/src/tests/briefResponse/training.ts]: might not be needed

File: ui-automated-tests/src/flows/briefResponse/training.ts [hunk 3]
```diff
@@ -1,28 +1,32 @@
...
 };
 
 const respond = async () => {
-  console.log("Starting to respond to atm brief");
+  console.log("respond to training");
+
+  await clickSubmitApplication();
 
-  await clickSubmitResponse();
   await util.matchText("a", "Enter a date for when you can start the project");
```
Comment (resh-ie) [inline ui-automated-tests/src/flows/briefResponse/training.ts]: needs to be changed to the new seller response

**Review by resh-ie — CHANGES_REQUESTED** [review]

File: ui-automated-tests/src/flows/brief/training.ts [hunk 4]
```diff
@@ -1,235 +1,135 @@
...
+  const sr = searchResult[0];
+  sr.click();
   await clickSaveContinue();
 };
 
 const fillWhoCanRespond = async () => {
-  await utils.selectRadio("allSellers");
   await clickSaveContinue();
+  await utils.matchText("li", "You must select at least one seller");
+  await selectDropBox();
```
Comment (resh-ie) [inline ui-automated-tests/src/flows/brief/training.ts:28]: No dropbox anymore =)

**Review by resh-ie — CHANGES_REQUESTED** [review]

File: ui-automated-tests/src/tests/briefResponse/training.ts (hunk 2 above)
Comment (malmas18) [inline ui-automated-tests/src/tests/briefResponse/training.ts]:
```suggestion
```

File: ui-automated-tests/src/flows/brief/training.ts (hunk 4 above)
Comment (malmas18) [reply ui-automated-tests/src/flows/brief/training.ts:28]: this is still there for selecting the seller

File: ui-automated-tests/src/flows/brief/training.ts [hunk 5]
```diff
@@ -1,235 +1,136 @@
...
-const fillBudgetRange = async () => {
-  await utils.type("input-budgetRange", { numberOfWords: 200 });
-  await clickSaveContinue();
-};
+  }
 
-const fillPaymentApproach = async () => {
-  await utils.type("input-paymentApproachAdditionalInformation", { numberOfCharacters: 200 });
-  await utils.selectRadio("fixedPrice");
+  await utils.type("working_arrangements", { numberOfCharacters: 150 });
```
Comment (tobiasbrown) [inline ui-automated-tests/src/flows/brief/training.ts]:
```suggestion
  await utils.type("working_arrangements", { numberOfWords: 150 });
```

File: ui-automated-tests/src/tests/briefResponse/training.ts [hunk 6]
```diff
@@ -2,30 +2,29 @@ import create from "../../flows/brief/training";
...
     await startBrief();
     await create({
-      evaluations: ["Interview", "References", "Case study", "Presentation"],
       locations: ["Australian Capital Territory", "Tasmania"],
       title,
     });
   });
 
-  it(`should be able to respond to training brief`, async () => {
+  it("should be able to respond training brief", async () => {
```
Comment (tobiasbrown) [inline ui-automated-tests/src/tests/briefResponse/training.ts]:
```suggestion
  it("should be able to respond to a training brief", async () => {
```

**Review by resh-ie — APPROVED** [review]

**Pull request merged**